packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -130,13 +144,17 @@ export class IamPolicyChecksWebview extends VueWebview {
                         },
                         (err, data) => {
                             if (err) {
-                                if (err.message.includes('The security token included in the request is invalid')) {
+                                if (
+                                    err.message.includes('The security token included in the request is invalid') ||
+                                    err.message.includes('The security token included in the request is expired')
```

Comment:
Can you maybe check for instances of `ExpiredTokenException` or similar?

Reply:
Yeah, that should do that trick for this instance. Unfortunately, I will have to parse it for the CLI output as we get it back as a blob.

Reply:
This does the trick for this instance, though I will still need to parse for the message for the CLI errors below, since they only give back string blobs.